middleware/scripts/ci.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash -e
```

Comment:
(optional) I'm not sure the whole Licence needs to be in a two-line ci script. :-)

Reply:
In b7e374be001d53bd17de21c56e191c56437ec288, I am merging in the upstream shiftdevices/base-middleware@6e07505 repo, so I'd prefer to make a minimum number of edits in the current PR. Let me know if you disagree with this approach.